init_files/boot_scripts/hello_world.sh

```diff
@@ -1,10 +0,0 @@
-#!/bin/bash
-
-################################################################
-#
-#  This Script can say 'Hello World'
-#  
-#  hello_world.sh -- Version 1.0
-################################################################
-
-echo 'Hello World. This is an example of a boot script!'
```

Comment:
These files might be nice to have in an "example boot scripts" directory, which is what they were intended for